projects/demo/src/modules/components/input-password/examples/2/index.ts

```diff
@@ -0,0 +1,32 @@
+import {Component} from '@angular/core';
+import {FormControl, FormGroup, Validators} from '@angular/forms';
+import {
+    TUI_INPUT_PASSWORD_DEFAULT_OPTIONS,
+    TUI_INPUT_PASSWORD_OPTIONS,
+} from '@taiga-ui/kit/components/input-password/input-password-options';
```

Comment:
It should be imported from the main entry point

Reply:
Done